src/category/ui/Add/CategoryInput.tsx

```diff
@@ -0,0 +1,90 @@
+import Icon from '@/common-ui/assets/Icon';
+import Input from '@/common-ui/Input';
+import Text from '@/common-ui/Text';
+import { theme } from '@/styles/theme';
+import getRem, { calculateRem } from '@/utils/getRem';
+import styled from '@emotion/styled';
+import React, { ReactNode } from 'react';
+
+interface CategoryInputProps {
+  children: ReactNode;
+}
+
+const CategoryInput = ({ children }: CategoryInputProps) => {
+  return <CategoryNameInputWrapper>{children}</CategoryNameInputWrapper>;
+};
+
+////////////////////////
+const CategoryNameInputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: ${getRem(20)};
+  padding: 0 ${getRem(20)};
+`;
+
+const CategoryNameInput = styled(Input)`
+  margin-top: ${getRem(20)};
+`;
+
+interface EmojiProps {
+  emoji: string;
+  setEmojiBSOpen: () => void;
```

Comment:
```suggestion
  onClickEmoji: () => void;
```

p3. 외부에서 정해준 동작의 의존적이라서 확장성이 조금 떨어지는 인터페이스인것같습니다ㅠㅠ
트리거 목적의 이벤트 핸들러이름으로 네이밍해주고 해당 핸들러에서 어떤 로직을 실행시킬지는 외부에서 정해주는것이 좋지않을까용?? 
사용하는 쪽 예시)

```tsx
<Emoji emoji="emoji" onClickEmoji={setEmojiBSOpen}/>
```